### PR TITLE
feat: Add k6 brower performance test

### PR DESCRIPTION
**Pull request by seakayone**
Checkout the included [README.md](https://github.com/dasch-swiss/dsp-das/pull/1666/files#diff-54a4915f496bff4a0e1ca9fbad867649bdd1c17931e405442c7d597289c60f5e) for info on these additions.

### CODE REVIEW

File: k6/pages/resourcepage.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export class Resourcepage {
```
Comment (derschnee68) [inline k6/pages/resourcepage.js:1]: ResourcePage typo ?

File: k6/pages/homepage.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export class Homepage {
```
Comment (derschnee68) [inline k6/pages/homepage.js:1]: HomePage?

**Review by derschnee68 — APPROVED** [review]

File: k6/pages/homepage.js (hunk 2 above)
Comment (seakayone) [reply k6/pages/homepage.js]: => [aca2fcb](https://github.com/dasch-swiss/dsp-das/pull/1666/commits/aca2fcbb539d8857f805304e9b2df8021f436101)

File: k6/pages/resourcepage.js (hunk 1 above)
Comment (seakayone) [reply k6/pages/resourcepage.js]: => [aca2fcb](https://github.com/dasch-swiss/dsp-das/pull/1666/commits/aca2fcbb539d8857f805304e9b2df8021f436101)

**Pull request merged**